internal/testprotos/make_protos.sh

```diff
@@ -4,15 +4,35 @@ set -e
 
 cd $(dirname $0)
 
+PROTOC_VERSION="3.5.1"
+PROTOC_OS="$(uname -s)"
+PROTOC_ARCH="$(uname -m)"
+case "${PROTOC_OS}" in
+  Darwin) PROTOC_OS="osx" ;;
+  Linux) PROTOC_OS="linux" ;;
+  *)
+    echo "Invalid value for uname -s: ${PROTOC_OS}" >&2
+    exit 1
+esac
+
+PROTOC="./protoc/bin/protoc"
+
+if [ "$(${PROTOC} --version 2>/dev/null)" != "libprotoc ${PROTOC_VERSION}" ]; then
```

Comment:
nit: prefer `[[` over `[` -- it has far more sane semantics regarding how tokens within get parsed

Reply:
Don't worry I have my own OCD things that I'm trying not to push onto this script :-)